src/main/java/seedu/address/logic/commands/ModCommand.java

```diff
@@ -2,17 +2,19 @@
 
 /**
  * Represents a command that modifies the mod attribute of the
- * person identified.
+ * batchmate identified.
  */
 public abstract class ModCommand extends Command {
 
     public static final String COMMAND_WORD = "mod";
     // TODO: Add the other mod commands
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Modifies the modules related to a person. "
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Modifies the modules related to a batchmate. "
             + "\nParameters: INDEX (must be a positive integer) MODULE [MORE_MODULES]..."
             + "\nExamples:"
-            + "\nmod add INDEX MODULE [MORE_MODULES]... : Adds the entered mods to the person at the index"
-            + "\nmod delete INDEX MODULE [MORE_MODULES]... : Deletes the entered mods from the person at the index";
+            + "\nmod add INDEX MODULE [MORE_MODULES]... : Adds the entered mods to the batchmate at the index."
+            + "\nmod delete INDEX MODULE [MORE_MODULES]... : Deletes the entered mods from the batchmate at the index."
+            + "\nmod mark INDEX MODULE [MORE_MODULES]... : Marks the entered mods of the batchmate "
```

Comment:
Maybe there should be a full stop here? :sweat_smile:
```suggestion
            + "\nmod mark INDEX MODULE [MORE_MODULES]... : Marks the entered mods of the batchmate. "
```

Reply:
Alright, added. Thanks! 😄